src/arena/string.rs

```diff
@@ -98,12 +98,12 @@ impl<'a> ArenaString<'a> {
         self.vec.is_empty()
     }
 
-    /// It's lengthy.
+    /// Its length.
     pub fn len(&self) -> usize {
         self.vec.len()
     }
 
-    /// It's capacatity.
+    /// Its capacity.
```

Comment:
I believe these ones are jokes :D

Reply:
I was trying to figure them out...

Reply:
Happy to remove. The tooling didn't even have an opinion about the first one, that was me.

The second one the tooling officially didn't like `capacatity` -- and I still don't understand the joke, but it's easy to tell the tooling to tolerate it.

Reply:
@lhecker: I've dropped the changes here. But could someone please explain the second joke? I'm embarrassed to admit that I don't get it...

Reply:
I'm making a joke about the pointlessness of documenting such obvious function names. If you look at it in the context all 3 of them are "It's" followed by the name of the function itself:

https://github.com/microsoft/edit/blob/b49c34f49347ac175ca88224873236f0c2722cda/src/arena/string.rs#L96-L109

(Hence also the spell error for capacity, to make it rhyme. 😅)